test/lib/autofix/snapshots/autofix.fixtures.ts.md

```diff
@@ -2263,3 +2206,1360 @@ Generated by [AVA](https://avajs.dev).
         warningCount: 0,
       },
     ]
+
+## General: jQueryModuleImports.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [
+          {
+            category: 1,
+            column: 24,
+            line: 5,
+            message: 'Unable to analyze this method call because the type of identifier "scrollbarWidth" in "jQuery.position.scrollbarWidth()"" could not be determined',
+          },
+        ],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'jQueryModuleImports.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: jQueryEnhancedModuleReplacement.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'jQueryEnhancedModuleReplacement.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+> AutofixResult: /jQueryEnhancedModuleReplacement.js
+
+    `sap.ui.define(["sap/ui/thirdparty/jquery", "sap/base/util/Version", "sap/ui/Device"], async function (jQuery, Version, Device) {␊
+    	// https://github.com/SAP/ui5-linter/issues/531␊
+    	var isLandscape = Device.orientation.landscape;␊
+    	var isPortait = Device.orientation.portrait;␊
+    	var isDesktop = Device.system.desktop;␊
+    	var isPhone = Device.system.phone;␊
+    	var isTablet = Device.system.tablet;␊
+    	var isAndroidPhone = Device.os.android && Device.system.tablet;␊
+    	var isAndroidTablet = Device.os.android && Device.system.tablet;␊
+    	var isIPhone = Device.os.ios && Device.system.phone;␊
+    	var isIPad = Device.os.ios && Device.system.ipad;␊
+    ␊
+    	// https://github.com/SAP/ui5-linter/issues/532␊
+    	var osName = Device.os.name;␊
+    	var osVersion = Version;␊
+    	var osVersionString = Device.os.versionStr;␊
+    	var isAndroid = Device.os.name === "Android";␊
+    	var isBlueBerry = Device.os.name === "bb";␊
+    	var isIOS = Device.os.name === "iOS";␊
+    	var isWinPhone = Device.os.name === "winphone";␊
+    	var isWin = Device.os.name === "win";␊
+    	var isLinux = Device.os.name === "linux";␊
+    	var isMac = Device.os.name === "mac";␊
+    ␊
+    	// https://github.com/SAP/ui5-linter/issues/588␊
+    	sap.ui.loader.config({paths: {"ui5.project.moduleA": "/ui5/project/moduleA"}});␊
+    	sap.ui.loader.config({paths: {"ui5.project.moduleB": "/ui5/project/moduleB"}});␊
+    ␊
+    	sap.ui.loader.config({paths: {"me/fancy/A": "ui5/projectA/"}});␊
+    	sap.ui.loader.config({paths: {"me/fancy/B": 'ui5/projectB/'}});␊
+    ␊
+    	// https://github.com/SAP/ui5-linter/issues/589␊
+    	var buttonPath1 = sap.ui.require.toUrl("sap.m.Button"); // resolves to "resources/sap/m/Button.js"␊
```

Comment:
This is not a correct replacement. The new API accepts a resource path (using slashes): https://ui5.sap.com/#/api/sap.ui%23methods/sap.ui.require.toUrl